tools/cosmovisor/cmd/cosmovisor/add_upgrade.go

```diff
@@ -24,6 +24,7 @@ func NewAddUpgradeCmd() *cobra.Command {
 
 	addUpgrade.Flags().Bool(cosmovisor.FlagForce, false, "overwrite existing upgrade binary / upgrade-info.json file")
 	addUpgrade.Flags().Int64(cosmovisor.FlagUpgradeHeight, 0, "define a height at which to upgrade the binary automatically (without governance proposal)")
+	addUpgrade.Flags().Bool(cosmovisor.FlagDisableRecase, false, "do not lower case plan name")
```

Comment:
Given that we get the cosmovisor config, I do not think we need a flag, we can get it directly from the cosmovisor config.